hydrolib/core/dflowfm/extold/models.py

```diff
@@ -290,23 +291,20 @@ def validate_quantity(cls, value):
         if isinstance(value, ExtOldQuantity):
             return value
 
-        def raise_error_tracer_name(quantity: ExtOldTracerQuantity):
+        def raise_error_prefix_name(quantity: str):
```

Comment:
can you remove this function it is being called once, so it does not save any encapsulation of lines